packages/components/src/Box/Box.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import classnames from "classnames";
+import { getBorderClassNames, getBorderVars } from "./utils/getBoxConfig";
+import {
+  getHeightClassName,
+  getHeightVars,
+  getWidthClassName,
+  getWidthVars,
+} from "./utils/getBoxDimension";
+import {
+  getMarginClassNames,
+  getMarginVars,
+  getPaddingClassNames,
+  getPaddingVars,
+} from "./utils/getBoxSpaces";
+import type { BoxProps } from "./Box.types";
+import styles from "./styles/Box.css";
+import gapStyles from "./styles/BoxGap.css";
+import radiusStyles from "./styles/BoxRadius.css";
+
+export function Box({
+  alignItems,
+  alignSelf,
+  as: Tag = "div",
+  background,
+  border,
+  borderColor,
+  children,
+  direction,
+  gap,
+  height = "auto",
+  justifyContent,
+  margin,
+  overflow,
+  padding,
+  position = "relative",
+  preserveWhiteSpace,
+  radius,
+  width = "auto",
+  ...props
+}: BoxProps) {
+  return (
+    <Tag
+      {...props}
+      className={classnames(
+        styles.box,
+        preserveWhiteSpace && styles["preserve-white-space"],
+        getPaddingClassNames(padding),
+        getMarginClassNames(margin),
+        getHeightClassName(height),
+        getWidthClassName(width),
+        getBorderClassNames(border),
+        radius && radiusStyles[`radius-${radius}`],
+        gap && gapStyles[`gap-${gap}`],
+      )}
+      style={{
+        ...getPaddingVars(padding),
+        ...getMarginVars(margin),
+        ...getHeightVars(height),
+        ...getWidthVars(width),
+        ...getBorderVars(border),
+        ...(background && { backgroundColor: `var(--color-${background})` }),
```

Comment:
Is it possible to restrict the usages to semantic tokens? With the new `--color-base` prefix on the "base" tokens that might make it easier to do automatically, but basically ensuring that "if you use an Atlantis component, it responds to theme/mode overrides" is something we can continue to confidently promise

Reply:
(Same note applies everywhere that we reference color in this file)